faq/manage-cluster-faq.md

```diff
@@ -35,16 +37,25 @@ By default, TiDB/PD/TiKV outputs standard error in the logs. If a log file is sp
 
 ### How to safely stop TiDB?
 
-Kill all the services using `kill` directly. The components of TiDB will do `graceful shutdown`.
+- If a load balancer is running (recommended): Stop the load balancer and execute the SQL statement `SHUTDOWN`. Then TiDB waits for a period as specified by [`graceful-wait-before-shutdown`](/tidb-configuration-file.md#graceful-wait-before-shutdown-new-in-v50) until all sessions are terminated. Then TiDB stops running.
+
+- If no load balancer is running: Execute the `SHUTDOWN` statement. Then TiDB components are gracefully stopped.
 
 ### Can `kill` be executed in TiDB?
 
-- You can `kill` DML statements. First use `show processlist` to find the ID corresponding with the session, and then run `kill tidb [session id]`.
-- You can `kill` DDL statements. First use `admin show ddl jobs` to find the ID of the DDL job you need to kill, and then run `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`. For more details, see the [`ADMIN` statement](/sql-statements/sql-statement-admin.md).
+- Kill DML statements:
+
+    First use `information_schema.cluster_processlist` to find the ID corresponding to the session, and then run the kill command.
+
+    TiDB v6.1.0 introduces the Global Kill feature (controlled by the `enable-global-kill` configuration, which is enabled by default). To enable Global Kill, execute `kill session_id`.
+
+    If the TiDB version is earlier than v6.1.0, or if the Global Kill feature is not enabled, `kill session_id` does not take effect by default. To terminate a DML statement, connect the client to the TiDB instance that is executing the DML statement and then execute the `kill tidb session_id` statement. If the client connects to another TiDB instance or if there is a proxy between the client and the TiDB cluster, the `kill tidb session_id` statement might be routed to other TiDB instances, which might incorrectly terminate other sessions. For details, see [`KILL`](/sql-statements/sql-statement-kill.md).
```

Comment:
```suggestion
    If the TiDB version is earlier than v6.1.0, or the Global Kill feature is not enabled, `kill session_id` does not take effect by default. To terminate a DML statement, the client must directly connect to the TiDB instance that is executing the DML statement and then execute the `kill tidb session_id` statement. If the client connects to another TiDB instance or there is a proxy between the client and the TiDB cluster, the `kill tidb session_id` statement might be routed to some other TiDB instance, which might incorrectly terminate other sessions. For details, see [`KILL`](/sql-statements/sql-statement-kill.md).
```

Reply:
1. Since we seldom use `must` and prefer active voice, how about using:

    `you should connect the client directly to the TiDB instance that is executing the DML statement.`

2. The grammar is not correct: some other instance. Can we just keep the original `other TiDB instances`?